src/web3/web3.erl

```diff
@@ -0,0 +1,40 @@
+-module(web3).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-behaviour(gen_server).
+-export([init/1, handle_call/3, handle_cast/2]).
+% TODO actually hook up to web3 XD
+
+%% how many coop chain blocks to wait before consider verified
+% -define(K, 10).
+
+%%
+init(_) ->
+  %?debugMsg("starting web3 server..."),
+  Pid = self(),
+  spawn_link(fun() -> height_poll("", Pid) end),
```

Comment:
I made the mistake of doing:
spawn_link(fun() -> height_poll("", self()) end),

Reply:
Lol, yeah, I've made that one too before :P